libs/perun/components/src/lib/members-list/members-list.component.ts

```diff
@@ -270,4 +275,17 @@ export class MembersListComponent implements OnChanges, AfterViewInit {
 
     this.dialog.open(MemberTreeViewDialogComponent, config);
   }
+
+  ngOnInit(): void {
+    if (this.includeConsents && this.facilityId) {
```

Comment:
I would argue that this should not be the responsibility of the table but rather the caller. In this case the `ResourceAssignedMembersComponent`.